packages/bruno-converters/src/utils.js

```diff
@@ -0,0 +1,190 @@
+const { customAlphabet } = require('nanoid');
+const each = require('lodash/each');
+const get = require('lodash/get');
+
+/**
+ * Generate a UUID
+ * @returns {string} UUID
+ */
+const uuid = () => {
+  // https://github.com/ai/nanoid/blob/main/url-alphabet/index.js
+  const urlAlphabet = 'useandom26T198340PX75pxJACKVERYMINDBUSHWOLFGQZbfghjklqvwyzrict';
```

Comment:
Why is this here? Shouldn't first line in this file be:

```javascript
const { customAlphabet, urlAlphabet } = require('nanoid');
```

It is exported by https://github.com/ai/nanoid/blob/c49cc113499f2b44ab52b80314e18043ad0f3f79/index.js#L5